### PR TITLE
Remove sensor-processing from rdx dependencies, moving up the height map renderer.

### PR DESCRIPTION
**Pull request by calvertdw**

### CODE REVIEW

File: ihmc-graphics/build.gradle.kts [hunk 1]
```diff
@@ -45,7 +45,8 @@ jmonkeyengineTestDependencies {
 }
 
 libgdxDependencies {
-   api(ihmc.sourceSetProject("main"))
```
Comment (calvertdw) [inline ihmc-graphics/build.gradle.kts:48]: This had the effect of bringing in sensor processing and the main source set, which brought in this stuff, too.
image

File: ihmc-graphics/build.gradle.kts [hunk 2]
```diff
@@ -45,7 +45,8 @@ jmonkeyengineTestDependencies {
...
+   api("us.ihmc:ihmc-java-toolkit:source")
+   api("us.ihmc:ihmc-robotics-toolkit:source")
```
Comment (calvertdw) [inline ihmc-graphics/build.gradle.kts:49]: This cuts out:
the main source set
ihmc sensor processing
ihmc communication
ihmc interfaces
ihmc robot models
vecmath
ihmc robot description
and this stuff
image
image

File: ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/ui/graphics/RDXHeightMapRenderer.java
Comment (calvertdw) [inline ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/ui/graphics/RDXHeightMapRenderer.java:1]: @PotatoPeeler3000 @LaughingCommit Would you guys prefer I keep this as a move or change it to a deletion? Are we done with this implementation?

**Review by ds58 — APPROVED** [review]

File: ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/ui/graphics/RDXHeightMapRenderer.java
Comment (calvertdw) [reply ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/ui/graphics/RDXHeightMapRenderer.java:1]: Talked to them and they want to keep it for now.

**Pull request merged**